packages/pwa-kit-react-sdk/src/ssr/universal/components/with-react-query/index.js

```diff
@@ -27,18 +29,32 @@ export const withReactQuery = (Wrapped, options = {}) => {
     /* istanbul ignore next */
     const wrappedComponentName = Wrapped.displayName || Wrapped.name
     const queryClientConfig = options.queryClientConfig
+    const beforeHydrate = options.beforeHydrate || passthrough
 
     /**
      * @private
      */
     class WithReactQuery extends FetchStrategy {
         render() {
+            let preloadedState = {}
+
             this.props.locals.__queryClient =
                 this.props.locals.__queryClient || new QueryClient(queryClientConfig)
 
+            if (!isServerSide) {
+                try {
+                    preloadedState = beforeHydrate(window.__PRELOADED_STATE__?.[STATE_KEY] || {})
+                } catch (e) {
+                    logger.error('Client `beforeHydrate` failed', {
+                        namespace: 'WithReactQuery render',
```

Comment:
Let's have the namespace match our convention...  more code like (e.g. no spaces, but can have dots). For example, see https://github.com/SalesforceCommerceCloud/pwa-kit/blob/777b7d6760579d13157bc60a7f196382462a42b8/packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js#L72

Reply:
I've updated the logger to use `with-react-query.render`